server/controllers/favorites.js

```diff
@@ -1,18 +1,18 @@
 const { default: mongoose } = require('mongoose');
-const Favorite = require('../models/Favorite.js')
+const favorite = require('../models/Favorite.js')
```

Comment:
This is actually one case (similar to React components) where it is preferred to capitalize the variable name. The reason being, just like React components, db models act kind of like blueprints to make copies of and use. So I'd recommend capitalizing `Favorite` but leaving all of the variables local to this file as starting with a lowercased first letter. Does that make sense?

Reply:
yes this makes sense! thank you